sass/core/_api.scss

```diff
@@ -305,3 +305,22 @@
   $val: _a_get($map, $val);
   @return _a_do-each($map, $val, $do);
 }
+
+
+// Error on Missing Keys
+// ---------------------
+/// A configuration option for setting the strictness with which `get-token()`
+/// handles missing nested keys. The option can be set to `'silent'`, `'null'`,
+/// `'warning'`, or `'error'`.
+///
+/// silent: The default state. `get-token()` will attempt to fail gracefully
+///         and greedily replace all existing references.
+/// null:   `get-token()` will return `null` if the value is not found,
+///         mimicking sass's inbuilt `map.get()` function.
+/// warn:  `get-token()` will return `null` if the value is not found, and
+///         also throw a warning.
+/// error: `get-token()` will throw an error if the value is not found.
+///
+/// @group core-get
+/// @type string
+$handle-missing-keys: 'silent' !default;
```

Comment:
This is the only user-facing change, right? The rest is internal performance optimization?

Reply:
Yep, just that config. Everything else is under the hood, and the current default is the existing behaviour. Should we throw an error if the value isn't one of the four possible options, or just allow it to carry out the default silent failure as it does currently?

Reply:
I suppose an error would be most clear and explicit - good call.

Reply:
Added the error, plus corresponding test. It's in the `_a_get()` function.